### DISCUSSION TITLE
Fitting a fibre orientations in the fundamental region (IPFs)

### DISCUSSION THREAD

**Discussion by M-Eddahbi** [mtex-toolbox/mtex#787] [Ask Anything]
Hi Rüdiger

I projected orientations axes on the fundamental region of an FCC material. I get the result shown in the attached figure. I would like to fit this projection to a fibre, and to determine the fibre axis as well.
I used the commands:
F=fibre(oriStart, oriStop)
F.r; % coordinates axis in sample reference
F.h; % coordinates axis in crystal reference

And I plot
- plotIPDF(F.symmetrise,r,'linewidth',2,'lineColor','orange'),

The figure is shown in attached image. As expected, the fitting is not good since only the initial and the end orientations are imputed.

The question is: 
Instead of two orientations only, how can we 
![Figure](https://user-images.githubusercontent.com/75380911/108200706-209cbd80-711f-11eb-8d6f-4f0ca71e1a75.jpg)
changed the command:
- plotIPDF([oriStart, oriStop], ref_direction) to fit all the orientations, or most of them?

Thank you in advance
Mohamed

**Comment by kilir** [comment]
Hi,
have a look at `fibre.fit()` In many cases, this should do what you want.

Cheers,
Rüdiger

**Reply by M-Eddahbi** [reply]
Thank you very much

**Reply by M-Eddahbi** [reply]
Hi Rüdiger,

Please, can you tell the corresponding link of   fibre.fit().

**Reply by kilir** [reply]
It's a function of fibre. See `help fibre.fit`.
Rüdiger

**Reply by M-Eddahbi** [reply]
Thank you

**Reply by M-Eddahbi** [reply]
Sorry for these inconveniences. I cannot found the above location help. Can you please precise more in details such location. 
Is it in Matlab help or in Mtex documentation?

**Reply by ralfHielscher** [reply]
Yes, you should type this into the matlab command window
```
help fibre.fit
```

Ralf.
PS: this works with any other command as well.

**Reply by M-Eddahbi** [reply]
Thank you Ralf
